scripts/ocsp-stapling2.test

```diff
@@ -416,7 +416,7 @@ remove_single_rF $ready_file5
                          -p $port5 -H loadSSL &
 server_pid5=$!
 wait_for_readyFile $ready_file5 $server_pid5 $port5
-echo "test connection" | openssl s_client -status -connect ${LOCALHOST}:$port5 -cert ./certs/client-cert.pem -key ./certs/client-key.pem -CAfile ./certs/ocsp/root-ca-cert.pem
+echo "test connection" | openssl s_client -status -legacy_renegotiation -connect ${LOCALHOST}:$port5 -cert ./certs/client-cert.pem -key ./certs/client-key.pem -CAfile ./certs/ocsp/root-ca-cert.pem
```

Comment:
Please add comment here about why this `-legacy_renegotiation` option is used and what is required in wolfSSL to remove it.